test/system/160-volumes.bats

```diff
@@ -319,5 +319,44 @@ EOF
     is "$output"  "" "no more volumes to prune"
 }
 
+# Named volumes copyup
+@test "podman volume create copyup" {
+    myvolume=myvol$(random_string)
+    mylabel=$(random_string)
+
+    # Create a named volume
+    run_podman volume create $myvolume
+    is "$output" "$myvolume" "output from volume create"
+
+    # Confirm that it shows up in 'volume ls', and confirm values
+    run_podman volume ls --format json
+    tests="
+Name           | $myvolume
+Driver         | local
+NeedsCopyUp    | true
+"
+    parse_table "$tests" | while read field expect; do
+        actual=$(jq -r ".[0].$field" <<<"$output")
+        is "$actual" "$expect" "volume ls .$field"
+    done
+
+    run_podman run --rm --volume $myvolume:/vol $IMAGE true
+    run_podman volume inspect --format '{{ .NeedsCopyUp }}' $myvolume
+    is "${output}" "true" "NeedsCopyUP should be true"
```

Comment:
The description is tautological. What is it about this test case that makes `NeedsCopyUp` be true, and could the description be changed to express that? E.g. `"volume .NeedsCopyUp after mounting on /vol"`. Similarly for the next one on line 349.

Reply:
Fixed.